src/main/java/com/ibs/kpitrackerservice/controller/UserController.java

```diff
@@ -1,7 +1,47 @@
 package com.ibs.kpitrackerservice.controller;
 
-import org.springframework.web.bind.annotation.RestController;
+import com.ibs.kpitrackerservice.model.User;
+import com.ibs.kpitrackerservice.service.UserService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
 
 @RestController
 public class UserController {
+
+    @Autowired
+    private UserService userService;
+
+    @GetMapping(path = "/user")
+    public ResponseEntity<List<User>> getAllUsers() {
+        return ResponseEntity.ok(userService.getAllUsers());
+    }
+
+    @PostMapping(path = "/user")
+    public ResponseEntity<String> saveUser(@RequestBody User user) {
+        return ResponseEntity.ok(userService.saveUser(user));
+    }
+
+    @PostMapping(path = "/user/update")
```

Comment:
Url pattern needs to be corrected by removing the *update* identifier from it. Moreover, the update operation should come under @PutMapping.